### PR TITLE
add serial number in the seat analysis tab

### PR DESCRIPTION
**Pull request by xbalajipge**
Added serial number in the **Seat Analysis** tab, see screen shot below

image

This is useful when the number of seats are 100+

_Not a **js** programmer, used GH copilot :)_

### CODE REVIEW

**Comment by xbalajipge** [comment]
Can this be approved and merged, thanks!

**Comment by xbalajipge** [comment]
who can approve these changes?

**Comment by karpikpl** [comment]
I think it's only @martedesco for now, I'll check with him if I can get merge permissions to help out with the repo.

**Comment by karpikpl** [comment]
@xbalajipge - I think we'll change it to "Seat Number" not "serial number"

File: src/components/SeatsAnalysisViewer.vue [hunk 1]
```diff
@@ -102,6 +103,7 @@ props: {
 data() {
     return {
         headers: [
+            { title: 'S.No', key: 'serialNumber'},
```
Comment (martedesco) [inline src/components/SeatsAnalysisViewer.vue:106]:
```suggestion
            { title: 'S.No', key: 'seatNumber'},
```

**Review by martedesco — APPROVED** [review]

**Pull request merged**